src/data_split.py

```diff
@@ -15,6 +15,10 @@ def df_to_dp(path):
     #path = '../data/utopia.txt' #for development
     #file = '/utopia.txt' #for development
     dp_path = '../data/datapackage'
+    # try:
```

Comment:
Question for me to learn: Why would you not use `try`?

Reply:
Good catch! That was a leftover from trouble shooting. I converted it back to a `try` `except`. Thanks!